examples/simple_script/BUILD

```diff
@@ -32,24 +36,36 @@ ruby_test(
     ],
 )
 
-# TODO: not the best way of defining this, should make `rubocop` rule like `buildifier`
-ruby_binary(
+ruby_rubocop(
     name = "rubocop",
-    srcs = [
-        ".relaxed-rubocop-2.4.yml",
-        ".rubocop.yml",
-        "script.rb",
-        "spec/script_spec.rb",
-        "@bundle//:bin/rubocop",
-    ],
-    args = [
-        "-c",
-        ".rubocop.yml",
-        "-P",
-        "-D",
-    ],
-    main = "@bundle//:bin/rubocop",
-    deps = [
-        "@bundle//:rubocop",
+    srcs = glob(
```

Comment:
This should be more like buildifier, where the sources are not needed to be defined. Not sure how it does that though.

Reply:
Totally, but I am focusing on mostly how can I expose each gem's folder so that I can feed it to pkg_zip ? pkg_zip already supports a prefix (like `vendor/bundle`) so all I need is to figure out how to pick gem contents out of the @bundle workspace, and then I am set. This is why I wanted to understand and refactor BUILD file generation for the @bundle and gems.